### PR TITLE
gdown: add page

### PR DESCRIPTION
**Pull request by sermelipharo**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 4.7.3

### CODE REVIEW

**Comment by spageektti** [comment]
Hi. Have you taken a look at contributing.md and the style guide?

**Comment by sermelipharo** [comment]
Yep, will fix now

**Comment by sermelipharo** [comment]
Fixed according guidelines

File: pages/common/gdown.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# gdown
+
+> Download files from Google Drive and other URLs.
+> Install: `pip install gdown`.
+> More information: <https://github.com/wkentaro/gdown>.
```
Comment (acuteenvy) [inline pages/common/gdown.md:5]: We do not document installation methods.
```suggestion
> Download files from Google Drive and other URLs.
> More information: <https://github.com/wkentaro/gdown>.
```

**Review by acuteenvy — CHANGES_REQUESTED** [review]

**Comment by sermelipharo** [comment]
Before this closes I want to express huge thanks to everyone for reviewing and helping with fixes and adapting! I am incredibly grateful to the community members for their help because this is my first pull request in "tldr", and I have limited experience with GitHub (as I am a graphic designer, mostly in print, not ui/digital, and I'm totally not a programmer). Your support really means a lot to me!

**Review by spageektti — APPROVED** [review]
LGTM, thanks for your contribution.

File: pages/common/gdown.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`gdown --fuzzy {{url}}`
+
+- Download a folder using its ID or the full URL:
+
+`gdown {{folder_id|url}} -O {{path/to/output_directory}} --folder`
+
+- Write stdout and pipe to extract:
+
+`gdown {{tar_url}} -O - --quiet | tar zxvf -`
```
Comment (acuteenvy) [inline pages/common/gdown.md:24]: `tar` will determine the compression method automatically. If you use the `z` option and try to extract a tarball compressed with something other than `gzip`, you'll get an error instead.
```suggestion
- Download a tar archive, write it to `stdout` and extract it:

`gdown {{tar_url}} -O - --quiet | tar xvf -`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**